File: python/recvosc.py
#!/usr/bin/python

from pythonosc import dispatcher
from pythonosc import osc_server

def test_handler(addr, val_i, val_f, val_s):
    print("{0} {1} {2} {3}".format(addr, val_i, val_f, val_s))

dispatcher = dispatcher.Dispatcher()
dispatcher.map("/test", test_handler)

server = osc_server.ThreadingOSCUDPServer(("0.0.0.0",7777), dispatcher)
server.serve_forever()

File: ruby/recvosc.rb
#!/usr/bin/ruby
require 'ruby-osc'

OSC.run do
  server = OSC::Server.new(7777)

  server.add_pattern "/test" do |*args|
    addr  = args[0]  # /test
    val_i = args[1]  # int
    val_f = args[2]  # float
    val_s = args[3]  # string

    puts "addr=#{addr}, val_i=#{val_i}, val_f=#{val_f}, val_s=#{val_s}"
  end
end
